vanilla-framework/vanilla-framework
# Example docs cleanup

## Done

- Tweaked some of the doc wording to remove passive tense
- Removed any hard-coded examples from markup files
- Created or copied over any examples as necessary

## QA

- Pull code
- Sanity check code
- Run `gulp jekyll`
- Run through all examples to make sure they are correct

## Details

Fixes #892 #893
